### ISSUE TITLE
set slot not work

### ISSUE THREAD

**Issue by UzumakiYT** [SkriptLang/Skript#1547]
```
command test:
	trigger:
		set slot 4 of player to book
```
work
```
command test:
	trigger:
		set slot 4 of player to book named "test"
```
not work, no errors

```
command test:
	trigger:
		set {_item} to book named "&ctest"
		set slot 4 of player to {_item}
```
work

2.3-alpha3

**Comment by GiraffeCubed** [comment]
I believe this was fixed in #1505

**Comment by TheBentoBox** [comment]
Yeah, I'd consider this a dupe. #1505 is the issue for item name issues.
